spec/routing/female_routing_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'prisoner routes', type: :routing do
+  describe get: '/prisons/PFI/prisoners/allocated' do
+    context 'with switch on' do
```

Comment:
This might be more descriptive:

```suggestion
    context 'with womens_estate switch on' do
```